Thumbprint/Targets/Thumbprint/Components/Button.swift

```diff
@@ -355,6 +345,10 @@ private extension Button {
             titleLabel.text = newValue
         }
         invalidateIntrinsicContentSize()
+
+        if accessibilityLabel == nil {
```

Comment:
The issue here is that if you update the title more than once you end up stuck with the first title for `accessibilityLabel`.

I think it would make more sense to still override `accessibilityLabel`, returning `super.accessibilityLabel ?? title` in the getter.

Reply:
Good call. Updated.